alf/algorithms/rl_algorithm.py

```diff
@@ -603,8 +604,17 @@ def _train_iter_off_policy(self):
                 with record_time("time/unroll"):
                     self.eval()
                     experience = self.unroll(unroll_length)
-                    self.summarize_rollout(experience)
-                    self.summarize_metrics()
+                    # The period of performing unroll may not be an integer
+                    # divider of config.summary_interval if config.unroll_length is not an
+                    # interger. In order to make sure the summary for unroll is
+                    # still written out about every summary_interval steps, we
+                    # need to remember whether summary has been written between
+                    # two unrolls.
+                    if self._need_to_summarize_rollout:
```

Comment:
This is never assigned as True?

Reply:
Oh, it's assigned in another local commit